relays/bin-substrate/src/cli/mod.rs

```diff
@@ -302,15 +290,48 @@ impl AccountId {
 	}
 }
 
-/// A trait representing an account address bound to a specific chain.
+/// Bridge-supported network definition.
 ///
-/// Can be used to convert [`AccountId`] formatting to be chain-specific.
-pub trait AccountChain {
-	/// Network name associated with the SS58 format.
-	const NAME: &'static str;
+/// Used to abstract away CLI commands.
+pub trait CliChain: relay_substrate_client::Chain {
+	/// Chain's current version of the runtime.
+	const RUNTIME_VERSION: sp_version::RuntimeVersion;
+
+	/// Crypto keypair type used to send messages.
+	///
+	/// In case of chains supporting multiple cryptos, pick one used by the CLI.
+	type KeyPair: sp_core::crypto::Pair;
+
+	/// Bridge Message Payload type.
+	///
+	/// TODO [ToDr] This should be removed in favour of target-specifc types.
```

Comment:
In case we have multiple bridges the `MessagePayload` type will be different.
For instance for `RialtoToWestend` and `RialtoToMillau` the `MessagePayload` and `encode_call` will look quite differently.
The TODO is there cause it's not the right way to abstract the code, however since right now we don't have a chain with two bridges that include MessageLanes it's good enough, but I already have an idea how to improve it, hence the TODO.